### PR TITLE
Redirect fileserver towards HTTPS

### PR DESCRIPTION
**Pull request by benjamreis**
Signed-off-by: BenjiReis <benjamin.reis@vates.fr>

### CODE REVIEW

File: SPECS/ocaml-xen-api-libs-transitional.spec [hunk 1]
```diff
@@ -233,6 +236,9 @@ touch %{build_ocaml_libdir}/xapi-libs-transitional/opam.config
 %{ocaml_libdir}/xapi-libs-transitional
 
 %changelog
+* Thu Dec 01 2022 Benjamin Reis <benjamin.reis@vates.fr> - 2.25.3-3.2
+- Add xen-api-libs-transitional-2.25.3-redirect-fileserver-https.patch
```
Comment (stormi) [inline SPECS/ocaml-xen-api-libs-transitional.spec]: suffix: .backport.patch

**Review by stormi — COMMENTED** [review]
In the commit message, s/filserver/fileserver/

File: SPECS/ocaml-xen-api-libs-transitional.spec (hunk 1 above)
Comment (benjamreis) [reply SPECS/ocaml-xen-api-libs-transitional.spec]: it's a new patch in another repo.
Do you still want the backport suffix?

**Review by gduperrey — APPROVED** [review]

File: SPECS/ocaml-xen-api-libs-transitional.spec [hunk 2]
```diff
@@ -18,6 +18,9 @@ Patch7: SOURCES/ocaml-xen-api-libs-transitional/0007-Receive-timeout-for-TCP-con
 Patch8: SOURCES/ocaml-xen-api-libs-transitional/0008-Total-timeout-for-receiving-HTTP-headers.patch
 Patch9: SOURCES/ocaml-xen-api-libs-transitional/0009-Maximum-header-length.patch
 
+# XCP-ng
+Patch1000: SOURCES/ocaml-xen-api-libs-transitional/xen-api-libs-transitional-2.25.3-redirect-fileserver-https.backport.patch
```
Comment (stormi) [inline SPECS/ocaml-xen-api-libs-transitional.spec]:
```suggestion
Patch1000: xen-api-libs-transitional-2.25.3-redirect-fileserver-https.backport.patch
```

Comment (stormi) [reply SPECS/ocaml-xen-api-libs-transitional.spec]: The prefix is both wrong and useless. Let's not copy this mistake (or convention without any meaning for us) from the rest of the spec file.

**Review by stormi — APPROVED** [review]

**Comment by stormi** [comment]
Koji build (v8.2-testing) http://koji.xcp-ng.org/taskinfo?taskID=26882

**Pull request merged**